### DISCUSSION TITLE
Messages missing a valid Message-ID when use gmail to send email

### DISCUSSION THREAD

**Discussion by tamdao** [supabase/supabase#27390] [Questions]
You will find a screenshot of the error message for your reference.
<img width="1017" alt="Screenshot 2024-06-19 at 23 18 06" src="https://github.com/supabase/supabase/assets/3086267/008a8a4c-1cb9-443f-ac07-1246e5ad47a4">

Could you please help me fix this issue? Any advice or recommendations you could offer would be greatly appreciated.

**Comment by GaryAustin1** [comment]
What is your SMTP email provider you are using to send the message?   Seems like it is not set up per the new methods required by Google for email it handles.

**Reply by tamdao** [reply]
I follow this guide https://github.com/orgs/supabase/discussions/5320#discussioncomment-7836793 to setup, but I got the error above
